paddle/fluid/framework/new_executor/pir_adaptor/pir_adaptor_util.cc

```diff
@@ -442,14 +500,14 @@ void HandleForSpecialOp(pir::Operation* op,
     auto value = op->result(0);
 
     value_exe_info->Add(value, param_name);
-  } else if (op_name == "builtin.constant") {
+  } else if (op->isa<pir::ConstantOp>()) {
```

Comment:
```suggestion
  } else if (op->isa<pir::ConstantTensorOp>()) {
```
分支里只有ConstantTensorOp逻辑，且if判断可以移除

Reply:
下个pr修改

Reply:
这里是我修改错误，ConstantTensorOp继承自ConstantOp，应该还用op_name判断，不能用isa判断，下个PR更改